docs/src/examples/components/Menu/Usages/MenuExampleToolbarWithSubmenu.shorthand.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react'
```

Comment:
In retrospect, I think just adding a more complex menu to the original example of **Menu as a Toolbar** would suffice. I don't think we need a new usage example for **Menu as a Toolbar with submenu**.

Reply:
Done.